receiver/otelarrowreceiver/config.go

```diff
@@ -18,22 +18,30 @@ type Protocols struct {
 	Arrow ArrowConfig             `mapstructure:"arrow"`
 }
 
+type AdmissionConfig struct {
```

Comment:
I think it would be good to update the README to properly reference the new configuration layout instead of the deprecated options.

Reply:
Thanks for the review! Updated the README